src/server/verification/verificationAPI.js

```diff
@@ -112,7 +112,21 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
             }
           }
         } else {
+          //only approved users can do the process
```

Comment:
Why we should deny user in the verification if it's pending in the queue ? 
Let it be verified, just won't whitelist him in the wallet.
On the DApp we could add special error screen for this case (user is verified but is pending) - "You're verified but you have to wait in the queue once you could be ready to claim. Please try again later..."

Reply:
bsically it shouldnt get here because on UI user wont be able to reach face verification
we dont want him to do the tedious face verification for nothing, ie he wont be able to claim after he is verified

Reply:
Okay, if it's just additional check, i think better to move it at the beginning of the `try {` block before we performing any verification actions (e.g. creating processor, validating, checking if user is already verified etc)

Reply:
Because now if user was already verified it will be whitelisted even it it's queue isn't ready yet

Reply:
if user already verified then there's no point in queue. verified users do not enter the queue again